Procfile

```diff
@@ -7,7 +7,7 @@ worker_buildapi_4hr: newrelic-admin run-program celery worker -A treeherder --wi
 worker_store_pulse_jobs: newrelic-admin run-program celery worker -A treeherder --without-gossip --without-mingle --without-heartbeat -Q store_pulse_jobs --concurrency=3
 worker_store_pulse_resultsets: newrelic-admin run-program celery worker -A treeherder --without-gossip --without-mingle --without-heartbeat -Q store_pulse_resultsets --concurrency=3
 worker_read_pulse_jobs: newrelic-admin run-program ./manage.py read_pulse_jobs
-worker_read_pulse_resultsets: newrelic-admin run-program ./manage.py read_pulse_resultsets
+worker_read_pulse_pushes: newrelic-admin run-program ./manage.py read_pulse_pushes
```

Comment:
Note: Heroku defaults 'new' worker types to zero dynos, so when this is deployed, the new entry will need bumping to 1 dyno, otherwise pulse pulse ingestion won't be running.